### PR TITLE
[#1401] improvement(dashboard): Fixed distorted page display due to excessively long property values

### PR DESCRIPTION
**Pull request by yl09099**
### What changes were proposed in this pull request?

Fixed the property value is too long and the page display is distorted.
1. If the content of the cell is too long, use ellipses to replace the excess.
2. Provide bubbles to show everything.

image
image

### Why are the changes needed?

Fix: #1401 

### Does this PR introduce _any_ user-facing change?

Yes.

### How was this patch tested?

Page display.

### CODE REVIEW

**Comment by yl09099** [comment]
@rickyma @maobaolong @xianjingfeng PATL.

File: dashboard/src/main/webapp/src/mock/coordinatorserverpage.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ Mock.mock(/\/coordinator\/conf/, 'get', {
     },
     {
       argumentKey: 'java.home',
-      argumentValue: '/home/hadoop/jdk-11.0.2'
+      argumentValue: '/root/jdk-11.0.2'
```
[the remark this answers is not in the document]
Comment (yl09099) [reply dashboard/src/main/webapp/src/mock/coordinatorserverpage.js]: There was a mistake in the substitution.fixed.

**Comment by maobaolong** [comment]
@yl09099 After https://github.com/apache/incubator-uniffle/pull/1951 this PR merged, it should not have unrelated System properties, do you get your screenshot after https://github.com/apache/incubator-uniffle/pull/1951 this PR?

**Comment by yl09099** [comment]
> @yl09099 After #1951 this PR merged, it should not have unrelated System properties, do you get your screenshot after #1951 this PR?

Understood, I may use the original version here, it is OK this PR does not affect, sometimes the path is too long will not look good, use this will not affect the beauty.

**Comment by maobaolong** [comment]
can I search a text within a long value after this PR？

**Comment by yl09099** [comment]
> can I search a text within a long value after this PR？

This was a very good suggestion and I decided to add a search window, which is done.

**Review by maobaolong — APPROVED** [review]
@yl09099 This PR looks pretty good, thanks for your contribution!

**Review by rickyma — APPROVED** [review]

**Comment by rickyma** [comment]
Merged. Thanks @yl09099 @advancedxy @maobaolong

**Pull request merged**